org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessParameterPanel.java

```diff
@@ -615,12 +615,129 @@ public boolean validateParameters() {
+					}
+				}
+				catch (Exception ex){}
+				
+				try {
+					valueMin_TS = new Timestamp(dateFormat.parse(field.getValueMin()).getTime());
+					value_TS = new Timestamp(dateFormat.parse(wEditor.getValue().toString()).getTime());
+					if(wEditor2!=null && wEditor2.getValue() != null) {
+						valueTo_TS = new java.sql.Timestamp(dateFormat.parse(wEditor2.getValue().toString()).getTime());
+					}
+				}
+				catch (Exception ex){}
+				
+				if(value_BD != null && valueMin_BD !=null) {
+					if(valueMin_BD.compareTo(value_BD) > 0) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" LessThanMinValue", field.getValueMin());
+						return false;
+					}
+					if(valueTo_BD!=null && valueMin_BD.compareTo(valueTo_BD) > 0) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" To LessThanMinValue", field.getValueMin());
+						return false;
+					}
+				}else if(value_TS != null && valueMin_TS != null){
+					if(value_TS.before(valueMin_TS)) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" LessThanMinValue", field.getValueMin());
+						return false;
+					}
+					if(valueTo_TS !=null && valueTo_TS.before(valueMin_TS)) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" To LessThanMinValue", field.getValueMin());
+						return false;
+					}
+					
+				}else {
+					if(field.getValueMin().compareTo(wEditor.getValue().toString()) > 0) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" LessThanMinValue", field.getValueMin());
+						return false;
+					}
+					if(wEditor2!=null && field.getValueMin().compareTo(wEditor2.getValue().toString()) > 0) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" To LessThanMinValue", field.getValueMin());
+						return false;
+					}
+				}
+			}// min value
+			
+			if(field.getValueMax() != null && wEditor.getValue() != null) {
+				BigDecimal valueMax_BD = null;
+				BigDecimal value_BD = null;
+				BigDecimal valueTo_BD = null;
+				
+				Timestamp valueMax_TS = null;
+				Timestamp value_TS = null;
+				Timestamp valueTo_TS = null;
+				try {
+					valueMax_BD = new BigDecimal(field.getValueMax());
+					value_BD = new BigDecimal(wEditor.getValue().toString());
+
+					if(wEditor2!=null && wEditor2.getValue() != null) {
+						valueTo_BD = new BigDecimal(wEditor2.getValue().toString());
+					}
+				}
+				catch (Exception ex){}
+				
+				try {
+					valueMax_TS = new Timestamp(dateFormat.parse(field.getValueMax()).getTime());
+					value_TS = new Timestamp(dateFormat.parse(wEditor.getValue().toString()).getTime());
+					if(wEditor2!=null && wEditor2.getValue() != null) {
+						valueTo_TS = new java.sql.Timestamp(dateFormat.parse(wEditor2.getValue().toString()).getTime());
+					}
+				}
+				catch (Exception ex){}
+				
+				if(value_BD != null && valueMax_BD !=null) {
+					if(valueMax_BD.compareTo(value_BD) < 0) {
+						Dialog.error(m_WindowNo, field.getColumnName()+" MoreThanMaxValue", field.getValueMax());
```

Comment:
MoreThanMaxValue and LessThanMinValue are translatable messages, you must parse correctly the message for the dialog